Pages/productlist.css

```diff
@@ -0,0 +1,124 @@
+.main-wrapper {
+  height: 90vh;
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: auto;
+  grid-template-areas:
+    "sidebar main main main main main"
+    "sidebar main main main main main";
+}
+.sidebar,
+.main {
+  padding: 1rem;
+}
+.sidebar {
+  grid-area: sidebar;
+  background-color: var(--Color_White);
+}
+.main {
+  grid-area: main;
+  display: grid;
+  gap: 20px;
+  background-color: var(--Primary_Color_100);
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+}
+
+/* Navbar section */
+.logo {
+  width: 100px;
+  max-width: 100%;
+}
+.navigation {
+  border: none;
+}
+.login-link {
+  color: var(--Color_White);
+}
+.secondary-list i {
+  cursor: pointer;
+}
+.secondary-list i:hover {
+  color: var(--Primary_Hover_Color);
+}
+
+/* Sidebar section */
+.filter-heading {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+  border-bottom: 1px solid var(--Primary_Color);
+}
+.filter-heading h3 {
+  font-size: 1.25rem;
+  font-family: "Barlow", sans-serif;
+}
+.clear-link {
+  cursor: pointer;
+  text-transform: uppercase;
+  font-family: sans-serif;
+  color: var(--Primary_Color);
+}
+.cat-heading {
+  padding: 10px 0;
+  font-family: "Barlow", sans-serif;
+}
```

Comment:
Suggestion: You can store the `font-family` in a variable and reuse it.

Rest all looks good.